submissions/edehJohn/html/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Gifted Hands Exquisite & Boutique</title>
+    </head>
+    <body>
+        <h1>Gifted Hands Exquisite & Boutique</h1>
+        <nav>
+            <ul>
+                <li><a href="#">Login</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Move to chart</a></li>
+            </ul>
+        </nav>
+            <h2><strong>Our Products and Services</strong></h2>
+            <ul>
+                <a href="#">Fashion & Design</a>
+                <ul>
+                    <img src="./img/Bride's-made-gown.jpeg" alt="My Gown"/>
```

Comment:
@john84edeh The images used in this file are not added to this pull request. Kindly update.

Reply:
AM GETTING CONFUSED HERE, HOW ARE MY SUPPOSE TO ADD THE IMAGES? AND HOW DO I DO A PULL REQUEST FOR THE IMAGES? I USED TWO IMAGES AS REQUESTED 
Bride's-made-gown.jpeg and pink-lace-rossy-buba. jpeg, CAN U PLEASE HELP ME OUT?